frontend/workflows/k8s/src/resize-hpa.tsx

```diff
@@ -49,6 +50,18 @@ const HPADetails: React.FC<WizardChild> = () => {
   const metadataAnnotations = [];
   const metadataLabels = [];
 
+  const [warnings, setWarnings] = React.useState<{
+    minSize: NoteConfig;
+    maxSize: NoteConfig;
+  }>({ minSize: undefined, maxSize: undefined });
+
+  const warningThreshold = 0.5;
```

Comment:
let's put constants like this in caps i.e. WARNING_THRESHOLD

Reply:
should I update the `alertLevel` variable as well?

Reply:
hm yeah sorry about that I probably should've done that. I guess I did that because it was inside the useEffect rather than outside. But yeah lets have both be all caps. Thanks!